packages/freezed/test/source_gen_src.dart

```diff
@@ -224,3 +224,30 @@ abstract class AstractClass {
 class _AstractClass implements AstractClass {
   const _AstractClass();
 }
+
+@ShouldThrow(
+  '@freezed classes configured with [finalize: true] must be sealed',
```

Comment:
Why is this necessary?

Reply:
Not marking the base class `sealed` more or less defeats the purpose of the entire change, because then the Dart analyzer can not figure out whether a pattern is "unmatchable".

However, from a technical perspective there is no need to seal the base class, so I guess it makes sense to remove that restriction

Reply:
Removed it